future_documentation.md

```
@@ -0,0 +1,7 @@
+currentPortfolio: Exact snapshot from the exchange (ReadOnly for UI).
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)